quizcon/main/models.py

```diff
@@ -187,6 +187,12 @@ def user_score(self):
         score = round(self.user_points() / self.quiz.total_points(), 2)
         return 0 if score < 0 else score
 
+    def user_score_percent(self):
+        """The user's total points divided by the possible score times 100"""
+        """All negative values are rounded up to 0"""
+        score = round(self.user_points() / self.quiz.total_points(), 2) * 100
```

Comment:
Is it possible for `self.quiz.total_points()` to ever return 0? If it is, then it might make sense to check for it to avoid a divide by 0 error.

Reply:
@nbuonin ah yes thank you! Fixed with new commit

Reply:
I think your initial logic was good. I meant if `self.quiz.total_points()` ever returns 0, then it would throw a divide by zero exception. If its not really possible for it to return 0, then your initial code is fine.

Reply:
@nbuonin Haa long day. self.quiz.total_points() will never equal zero!